packages/web/src/components/GraphQLHooksProvider.tsx

```diff
@@ -2,6 +2,9 @@ import type { DocumentNode } from 'graphql'
 
 export interface GraphQLHookOptions {
   variables?: Record<string, any>
+  refetchQueries?: { query: DocumentNode }[]
+  onCompleted?: () => any
```

Comment:
```suggestion
  onCompleted?: (data: any) => void
```

Reply:
I know this is not perfect at the moment, I'll have to take a deeper look into typing this more completely